packages/cozy-harvest-lib/src/helpers/daytime.js

```diff
@@ -11,8 +11,8 @@ export const randomDayTime = (
   end = 1,
   randomize = (min, max) => Math.random() * (max - min) + min
 ) => {
-  if (!isInteger(start)) throw new Error('Parameter start must be an integer')
-  if (!isInteger(end)) throw new Error('Parameter end must be an integer')
+  if (!isNumber(start)) throw new Error('Parameter start must be an integer')
+  if (!isNumber(end)) throw new Error('Parameter end must be an integer')
```

Comment:
[nit] The error message should reflect that: s/an integer/a number/

Reply:
Oh thanks, I am a little bit tired.